libvast/src/format/zeek.cpp

```diff
@@ -475,6 +479,9 @@ caf::error reader::parse_header() {
   }
   // Add #index=hash attribute for fields where it makes sense.
   add_hash_index_attribute(layout_);
```

Comment:
Should the `#index=hash` attributes be reflected in the schema instead?

Reply:
I think they should, but it still makes sense to add this attribute for fields that are not part of the user defined schema. In the current implementation, one would have to supply `#index=default` to avoid these fields to automatically get the hash index.